.ci/opensearch/Dockerfile

```diff
@@ -1,10 +1,18 @@
-ARG OPENSEARCH_VERSION
+ARG OPENSEARCH_VERSION=latest
 FROM opensearchproject/opensearch:$OPENSEARCH_VERSION
 
+# OPENSEARCH_VERSION needs to be redefined as any arg before FROM is outside build scope.
+# Reference: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
+ARG OPENSEARCH_VERSION=latest
 ARG opensearch_path=/usr/share/opensearch
 ARG opensearch_yml=$opensearch_path/config/opensearch.yml
 
 ARG SECURE_INTEGRATION
 RUN echo "plugins.ml_commons.only_run_on_ml_node: false" >> $opensearch_yml;
 RUN echo "plugins.ml_commons.native_memory_threshold: 100" >> $opensearch_yml;
+RUN if [ "$OPENSEARCH_VERSION" == "2.11.0" ] ; then \
```

Comment:
may be we can try to do something like: `"$OPENSEARCH_VERSION" >= "2.8.0"`, so that we don't have to change this file everytime we upgrade version

Reply:
So, that is not going to be straight forward. Reason being that its not a simple string comparison.    
Example:
```
"2.8.0" is actually greater than "2.11.0"
```

If we want to do this, we probably need to split our versions into Major, Minor and Patch .. aka X, Y and Z and then compare individually.

Reply:
Another way to handle this is to have these settings update via `opensearch-py` client. we can update settings using  `put_settings` method. 
https://github.com/opensearch-project/opensearch-py/blob/da436cbbe8dda34abd607f527d4f0bdacb9b30d8/opensearchpy/client/cluster.py#L292